secprofile/oci_profile.go

```diff
@@ -9,6 +9,22 @@ import (
 	"github.com/opencontainers/runtime-spec/specs-go"
 )
 
+type APIID string
+
+type APISubsetId string
+
+type APIAccess string
+
+const (
+	Allow  APIAccess = "allow"
+	Access APIAccess = "access"
```

Comment:
Why do we need `APIAccess="Access"`?

Reply:
hmm I only remember us deciding on:
```go
Access APIAccess = "access"
Deny APIAccess = "deny"
```

Reply:
Why adding `Access APIAccess = "access"` ?

Also for the type, it's better to have a different type when it is enumerable.

Reply:
I believe this is a typo from our pairing session. I'll keep Allow and Deny, and remove the third line

Reply:
https://github.com/n4ss/libentitlement/commit/671f14ba9ef8489342a82166c0e65c7c09ebbba1#diff-68edf18e069363d58649944c4d3ec6fbR17

LGTM for Access/Deny! ^_^